File: 0128. Longest Consecutive Sequence/Solution2.java
import java.util.*;

class Solution {
    public int longestConsecutive(int[] nums) {
        if(nums.length == 0) return 0;
        
        TreeSet<Integer> set = new TreeSet<>();
        int longest=1,count=1,index=-1;

        for(int i: nums) { set.add(i); }
        int[] list = new int[set.size()];
        // System.out.print(list.length);
        for(int i: set) { list[++index] = i; }

        // for(int i: list) System.out.print(i + " ");

        for(int i=0;i<list.length-1;i++) {
            if(list[i+1]-list[i] == 1) {
                count++;
                System.out.println(count);
            }
            else {
                longest = Math.max(longest, count);
                count = 1;
            }
            longest = Math.max(longest, count);
        }
        return longest;
    }
}
public class Solution2 {
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int[] nums = new int[n];
        for(int i=0;i<n;i++) nums[i] = sc.nextInt();
        Solution sol = new Solution();
        System.out.println(sol.longestConsecutive(nums));
    }
}

File: 1512. Number of Good Pairs/solution2.c
#include<stdio.h>

int numIdenticalPairs(int* nums, int n){
    int count=0;
    for(int i=0;i<n-1;i++)
    {
        for(int j=i+1;j<n;j++)
        {
            if(nums[i]==nums[j])
            count++;
        }
    }
    return count;
}
int main()
{
    int nums[] = {1,2,3,1,1,3};
    printf("%d", numIdenticalPairs(nums, 6));
}